resources/ttReportScripts_new.jsp

```diff
@@ -1,3609 +1,3626 @@
-                                divBlockTwo += "</div>";
-                            }
-                            i++;
-                        });
-                        containerConvas = containerConvas + divBlockOne + divBlockTwo + "</div>"
-                    }
-                    return "<i id='emotioniconID"+row[0]+"' style='cursor:pointer;' class='fa fa-heart' aria-hidden='true' onclick='loadEmotionMap("+row[0]+");'></i>"+containerConvas;
-                }
-            },
-                
-    	]
-    }    
-    );
-    }
-    else {
-        perStudentReport  =  $('#perStudentReport').DataTable({
-            data: [],
-            destroy: true,
-            columns: [
-                { title: headers['sid'] },
-                { title: headers['sname']  },
-                { title: headers['uname']  },
-                { title: headers['problems']  },
-                { title: headers['effchart']  },
-                { title: headers['emochart']  },
-            ],
-            "bPaginate": false,
-            "bFilter": false,
-            "bLengthChange": false,
-            rowReorder: false,                
-            "scrollX": true,
-            "bSort" : false,
-            "columnDefs": [
-                {
-                    "width": "5%",
-                    "targets": [ 0 ],
-                    "visible": false
-
-                },{
-                    "width": "5%",
-                    "targets": [ 1 ],
-                    "visible": true
-
-                },{
-                    "width": "5%",
-                    "targets": [ 2 ],
-                    "visible": true
-
-                },
-                {
-                    "width": "5%",
-                    "targets": [ 3 ],
-                    "visible": true,
-                    'className': 'dt-body-center',
-                    'render': function ( data, type, row ) {
-                        return '<label>'+data+'&nbsp&nbsp</lable><a  class="viewEachStudentDetail" aria-expanded="true" aria-controls="collapseOne"><i class="glyphicon glyphicon-menu-down"></i></a>';
-                    }
-
-                },
-                {
-                    "targets": [ 4 ],
-                    "width": "10%",
-                    'className': 'dt-body-center',
-                    'render': function ( data, type, row ) {
-                        var effortChartId = "effortChart"+row[0];
-                        var containerChart = "containerChart"+row[0];
-                        var legendChart = "legendChart"+row[0];
-                        var dataContent = "<div id="+containerChart+" style='width:900px;height:500px;display:none'><div class='panel panel-default'><div class='panel-heading'>"+headers['effchart']+"</div><div class='panel-body'><canvas width='800' height='150' id="+effortChartId+"></canvas></div><div class='panel-body' id='"+legendChart+"'></div></div></div>";
-                        return "<i id='iconID"+row[
```

Comment:
Double check this file as The whole file is coming in the commit as changed

Reply:
This file also got converted to unix eol format.  No substantive changes for issue 21. I will replace the whole file contents.